docs/deployment.md

```diff
@@ -1,48 +1,67 @@
-[Kubernetes](https://github.com/jaegertracing/jaeger-kubernetes)
-and [OpenShift](https://github.com/jaegertracing/jaeger-openshift).
+Component             | Repository
+--------------------- | ---
+**jaeger-agent**      | [https://hub.docker.com/r/jaegertracing/jaeger-agent/](https://hub.docker.com/r/jaegertracing/jaeger-agent/)
+**jaeger-collector**  | [https://hub.docker.com/r/jaegertracing/jaeger-collector/](https://hub.docker.com/r/jaegertracing/jaeger-collector/)
+**jaeger-query**      | [https://hub.docker.com/r/jaegertracing/jaeger-query/](https://hub.docker.com/r/jaegertracing/jaeger-query/)
+
+There are orchestration templates for running Jaeger with:
+
+  * Kubernetes: [https://github.com/jaegertracing/jaeger-kubernetes](https://github.com/jaegertracing/jaeger-kubernetes),
+  * OpenShift: [https://github.com/jaegertracing/jaeger-openshift](https://github.com/jaegertracing/jaeger-openshift).
 
 ## Agent
 
-Jaeger client libraries expect `jaeger-agent` process to run locally on each host.
-The agent exposes the following ports: `5775/udp 6831/udp 6832/udp 5778`.
+Jaeger client libraries expect **jaeger-agent** process to run locally on each host.
+The agent exposes the following ports:
+
+Port | Protocol | Function
+---- | -------  | ---
+5775 | UDP      | accept zipkin.thrift over compact thrift protocol
+6831 | UDP      | accept jaeger.thrift over compact thrift protocol
+6832 | UDP      | accept jaeger.thrift over binary thrift protocol
+5778 | HTTP     | serve configs, sampling strategies
 
 ### Discovery System Integration
 
 The agents can connect point to point to a single collector address, which could be
-load balanced by another infrastructure component across multilpe collectors. The agent
-can also be configured with a static list of collector addresses.
+load balanced by another infrastructure component (e.g. DNS) across multilpe collectors.
+The agent can also be configured with a static list of collector addresses.
 
-In the future we will support using a discovery system to dynamically load balance
-across several collectors. We use [go-kit](https://github.com/go-kit/kit) to support
-a number of different discovery systems, but currently blocked by [Issue 492](https://github.com/go-kit/kit/pull/492).
+In the future we will support different service discovery systems to dynamically load balance
+across several collectors ([issue 213](https://github.com/uber/jaeger/issues/213)).
 
 ## Collectors
 
-Collectors require almost no configuration except for the location of Cassandra cluster.
+Many instances of **jaeger-collector** can be run in parallel.
+Collectors require almost no configuration, except for the location of Cassandra cluster,
+via `-cassandra.keyspace` and `-cassandra.servers` options.
 All configuration parameters can be provided via command line options.
-At default settings the collector exposes the following ports: `14267 14268`.
 
 ```go
 go run ./cmd/collector/main.go -h
```

Comment:
Would it be better to use:
`docker run -it --rm --network=host jaegertracing/jaeger-collector /go/bin/collector-linux -h`
so does not require user to run a Golang app?